packages/repository/test/unit/repository-mixin/repository-mixin.test.ts

```diff
@@ -10,67 +10,65 @@ import {
   DataSourceConstructor,
   Class,
 } from '../../../';
-import {Application, Component} from '@loopback/core';
-
-// tslint:disable:no-any
+import { Application, Component, Server } from '@loopback/core';
```

Comment:
The changes made in this file seem unrelated to the rest of this pull request, could you please revert them? If it's an important cleanup, then please open a new PR for that.